src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -301,6 +302,42 @@ public static void LogHttpDependency(this ILogger logger, HttpRequestMessage req
             logger.LogInformation(HttpDependencyFormat, targetName, dependencyName, (int) statusCode, duration, startTime.ToString(CultureInfo.InvariantCulture), isSuccessful, context);
         }
 
+        /// <summary>
+        ///     Logs a SQL dependency
+        /// </summary>
+        /// <param name="logger">Logger to use</param>
+        /// <param name="connectionString">SQL connection string</param>
+        /// <param name="tableName">Name of table</param>
+        /// <param name="operationName">Name of the operation that was performed</param>
+        /// <param name="isSuccessful">Indication whether or not the operation was successful</param>
+        /// <param name="measurement">Measuring the latency to call the SQL dependency</param>
+        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        public static void LogSqlDependency(this ILogger logger, string connectionString,  string tableName, string operationName, DependencyMeasurement measurement, bool isSuccessful, Dictionary<string, object> context = null)
+        {
+            Guard.NotNull(logger, nameof(logger));
+            Guard.NotNull(measurement, nameof(measurement));
+
+            LogSqlDependency(logger, connectionString, tableName, operationName, measurement.StartTime, measurement.Elapsed, isSuccessful, context);
+        }
+
+        /// <summary>
+        ///     Logs a SQL dependency
+        /// </summary>
+        /// <param name="logger">Logger to use</param>
+        /// <param name="connectionString">SQL connection string</param>
+        /// <param name="tableName">Name of table</param>
+        /// <param name="operationName">Name of the operation that was performed</param>
+        /// <param name="startTime">Point in time when the interaction with the HTTP dependency was started</param>
+        /// <param name="duration">Duration of the operation</param>
+        /// <param name="isSuccessful">Indication whether or not the operation was successful</param>
+        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        public static void LogSqlDependency(this ILogger logger, string connectionString,  string tableName, string operationName, DateTimeOffset startTime, TimeSpan duration,  bool isSuccessful, Dictionary<string, object> context = null)
+        {
```

Comment:
Should we guard for empty connection string or does the builder handle that?

Reply:
Hmm, the connection string builder doesn't guard against null or empty. Maybe we should handle that.

Reply:
Think that's a good idea!

Reply:
Done.